balboa-http/src/main/scala/com/socrata/balboa/server/Main.scala

```diff
@@ -2,47 +2,79 @@ package com.socrata.balboa.server
 
 import javax.servlet.http.HttpServletRequest
 
-import com.socrata.balboa.metrics.config.Configuration
-import com.socrata.balboa.server.rest.{EntitiesRest, MetricsRest, VersionRest}
-import com.socrata.http.routing.{HttpMethods, SimpleRoute, SimpleRouter}
-import com.socrata.http.server.implicits._
-import com.socrata.http.server.responses._
-import com.socrata.http.server.{Filter, HttpResponse, Service, SimpleFilter, SocrataServerJetty}
+import com.socrata.balboa.BuildInfo
+import com.socrata.balboa.metrics.data.BalboaFastFailCheck
+import com.socrata.balboa.server.rest.{EntitiesRest, MetricsRest}
 import com.typesafe.scalalogging.slf4j.StrictLogging
-import org.apache.log4j.PropertyConfigurator
+import org.eclipse.jetty.server.Server
+import org.scalatra.{InternalServerError, NotFound, Ok, ScalatraServlet}
+import org.slf4j.LoggerFactory
+import org.eclipse.jetty.webapp.WebAppContext
+import org.json4s.{DefaultFormats, Formats}
+import org.scalatra.servlet.ScalatraListener
+import org.scalatra.json.JacksonJsonSupport
 
-class Main
+import scala.collection.JavaConverters._
 
-object Main extends App with StrictLogging {
-  val DefaultPort = 9012
+class MainServlet extends ScalatraServlet with StrictLogging with JacksonJsonSupport {
```

Comment:
This can go in it's own file. Not required for Scala, but this is big enough to make it worth while. It's generally easier to find things where the class name corresponds to the file name.